app/2.0/docs/tools/create-element-polymer-cli.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Create an element project with the Polymer CLI
+---
+
+<!-- toc -->
+
+This section shows you how to start an element project.
+
+1.  Create a directory for your element project. It's best practice for the name
+    of your project directory to match the name of your element.
+
+    <pre><code>mkdir <var>my-el</var> && cd <var>my-el</var></code></pre>
+
+    Where <code><var>my-el</var></code> is the name of the element you're
+    creating.
+
+1.  Initialize your element. Polymer CLI asks you a few
+    questions as it sets up your element project.
+
+        polymer init
+
+1.  Select `polymer-2-element`.
+
+1.  Enter a name for your element.
+
+    The [custom elements
+    specification](https://www.w3.org/TR/2016/WD-custom-elements-20160226/#concepts) requires the
+    element name to contain a dash.
+    {.alert .alert-info}
+
+1.  Enter a description of the element.
+
+At this point, Polymer CLI generates files and directories for your element, and installs your 
+project's dependencies.
+
+## Element project layout
+
+After the initialization process Polymer CLI generates the following files and directories.
+
+*   `bower.json`. Configuration file for Bower.
+*   `bower_components/`. Project dependencies. See [Manage dependencies](#dependencies).
+*   `demo/index.html`. Demo of <code><var>my-el</var></code>`.html`.
+*   `index.html`. Automatically-generated API reference.
+*   <code><var>my-el</var></code>`.html`. Element source code.
+*   `test/`<code><var>my-el</var></code>`_test.html`. Unit tests for
+    the element.
+
+## Manage dependencies and HTML imports {#dependencies}
+
+### Use Bower to manage dependencies
+
+Polymer CLI uses [Bower](http://bower.io) for dependency management.
+
+Dependencies are stored in the `bower_components` directory. You should never manually alter the
+contents of this directory.
+
+Use the Bower CLI to manage dependencies.
+
+This command downloads a dependency to `bower_components/` (the `--save` flag saves the new 
```

Comment:
Rather than "This command" form, maybe replace these with:

To add a new Bower dependency to your project:

    <bower command>

As Fred suggested, "This command..." is a little ambiguous about whether you're talking about the command above, or the command below.